src/app/configuration/resources/recaptcha-key-injectable.ts

```diff
@@ -0,0 +1,10 @@
+import {InjectionToken} from '@angular/core';
+
+export const RECAPTCHA_KEY_INJECTABLE = new InjectionToken<string>(
+  'Recaptcha key',
+  {
+    factory: () => {
+      return '6LeeO6QaAAAAAHcFrYzi2c-sxZjUYWifhdpKpFiR';
```

Comment:
Mozno by nebolo odveci mat toto tiez v nasom config file. A tahat to tak cez resource servicu.